packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -76,20 +76,11 @@ function ButtonBlockAppender(
 						// eslint-disable-next-line no-restricted-syntax
 						disabled={ disabled }
 						label={ label }
+						showTooltip
 					>
-						{ ! hasSingleBlockType && (
-							<VisuallyHidden as="span">{ label }</VisuallyHidden>
-						) }
```

Comment:
I think this was a leftover when the button appender had a visual label. The button has an `aria-label`, so visually hidden text shouldn't be necessary for a11y.

cc @afercia